### PR TITLE
Velocity Control

### PR DESCRIPTION
**Pull request by KenwoodFox**
Test before merging.

Creates a new method in `Drivetrain.java` that allows us to drive the robot using a value in M/s (from ROS).

### CODE REVIEW

File: src/main/java/frc/robot/commands/ROSControl.java [hunk 1]
```diff
@@ -31,6 +30,6 @@ public ROSControl(Drivetrain _drivetrain, ROS _ros) {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    drivetrain.FlyWithWiresA(ros.getStarboardSpeed(), ros.getPortSpeed());
+    drivetrain.FlyWithWiresB(ros.getStarboardSpeed(), ros.getPortSpeed());
```
Comment (boinary) [inline src/main/java/frc/robot/commands/ROSControl.java]: Why change from A to B?

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 2]
```diff
@@ -122,6 +122,22 @@ public void FlyWithWiresA(double starboard, double port){
...
+   */
+  public void FlyWithWiresB(double starboard_meters, double port_meters){
+    // Convert the M/s into M/10ms, then, convert M/10ms into Ticks/10ms
+    double starboard_miliseconds = ((starboard_meters / 100) * Constants.DrivetrainPID.ticksPerMeter);
+    double port_miliseconds = ((port_meters / 100) * Constants.DrivetrainPID.ticksPerMeter);
+
+    starboardMotor.set(ControlMode.Velocity, starboard_miliseconds);
+    portMotor.set(ControlMode.Velocity, port_miliseconds);
+  }
+
```
Comment (boinary) [inline src/main/java/frc/robot/subsystems/Drivetrain.java]: I thought the RIO is not this precise?

File: src/main/java/frc/robot/commands/ROSControl.java (hunk 1 above)
Comment (KenwoodFox) [reply src/main/java/frc/robot/commands/ROSControl.java]: Mode B is velocity control, ROS needs to use Velocity control.

File: src/main/java/frc/robot/subsystems/Drivetrain.java (hunk 2 above)
Comment (KenwoodFox) [reply src/main/java/frc/robot/subsystems/Drivetrain.java]: Im not sure what you mean by that, these are being handed to the motor controllers.

Comment (boinary) [reply src/main/java/frc/robot/subsystems/Drivetrain.java]: My bad

Comment (KenwoodFox) [reply src/main/java/frc/robot/subsystems/Drivetrain.java]: No worries :)

Comment (boinary) [reply src/main/java/frc/robot/subsystems/Drivetrain.java]: This looks good. I see no issues

**Pull request merged**